README.md

```diff
@@ -16,17 +16,19 @@ A curated list of awesome public dbt projects that the community can learn from.
 
 ## Projects
 
-- [Jaffle Shop](https://github.com/dbt-labs/jaffle_shop) - The canonical dbt sample project. ([with semantic layer](https://github.com/infuseai/jaffle-sl-template)
+- [Jaffle Shop](https://github.com/dbt-labs/jaffle_shop) - The canonical dbt sample project. ([with semantic layer](https://github.com/infuseai/jaffle-sl-template).
 - [Spellbook](https://github.com/duneanalytics/spellbook) - Blockchain data analytics.
 - [Filecoin Data](https://github.com/davidgasquez/filecoin-data-portal) - Filecoin data portal.
 - [CountMoney](https://github.com/flyanakin/CountMoney) - Finance data pipeline orchestration.
 - [Danish Parliament Data](https://github.com/bgarcevic/danish-democracy-data) - Danish Parliament data from Folketinget API.
-- [Taiwan Campaign Finance](https://github.com/g0v/tw_campaign_finance) - Campaign finance analytics with duckdb and [Mosaic Vgplot](https://uwdata.github.io/mosaic/vgplot/)
-- [Bird Data](https://github.com/dagster-io/mdsfest-opensource-mds) - Bird Data with [Dagster](https://dagster.io/), [Evidence](https://evidence.dev/) and duckdb
-- [Datadex](https://github.com/davidgasquez/datadex) - Climate & blockchain data modeling.
+- [Taiwan Campaign Finance](https://github.com/g0v/tw_campaign_finance) - Campaign finance analytics with duckdb and [Mosaic Vgplot](https://uwdata.github.io/mosaic/vgplot/).
+- [Bird Data](https://github.com/dagster-io/mdsfest-opensource-mds) - Bird Data with [Dagster](https://dagster.io/), [Evidence](https://evidence.dev/) and duckdb.
+- [Datadex](https://github.com/davidgasquez/datadex) - Generic local friendly dbt project optimized for open data.
```

Comment:
Let me know if this description doesn't sounds good! Happy to change it. Wanted to make it more abstract as the project changes quite a bit on the data it uses. :sweat_smile: